server/datastore/mysql/queries.go

```diff
@@ -2,12 +2,65 @@ package mysql
 
 import (
 	"database/sql"
+	"fmt"
 
 	"github.com/jmoiron/sqlx"
 	"github.com/kolide/fleet/server/kolide"
 	"github.com/pkg/errors"
 )
 
+func (d *Datastore) ApplyQueries(authorID uint, queries []*kolide.Query) (err error) {
+	tx, err := d.db.Begin()
+	if err != nil {
+		return errors.Wrap(err, "begin ApplyQueries transaction")
+	}
+
+	defer func() {
```

Comment:
why not move this to right after the call to tx.Commit?

Reply:
We need to call `Rollback` if any of the error cases occur between here and the call to `Commit`. This is becoming a common pattern, and I'm tempted to abstract into a function in Kolide Kit.